pymanopt/manifolds/psd.py

```diff
@@ -79,6 +79,9 @@ def retr(self, X, G):
         return self.exp(X, G)
 
     def norm(self, x, u):
+        # This implementation is as fast as np.linalg.solve_triangular and is
+        # more stable, as the solver tends to output non positive definite
```

Comment:
as the solver --> as the above

(to avoid confusion -- I'd change it myself, but didn't find the option on github interface, sorry :/)